src/phrases/phrases-en.ts

```diff
@@ -1414,11 +1414,11 @@ const phrases = {
       source: {
         title: 'Source version',
         subTitle: 'Base this version on an existing version',
-        defaultOption: 'N/A',
+        defaultOption: 'Draft',
       },
       locked: {
         title: 'Locked',
-        subTitle: 'Is the version locked:',
+        subTitle: 'Lock this version to disable deletion',
```

Comment:
Pirk. Synes dette er litt tydeligere.
```suggestion
        subTitle: 'Lock this version to prevent deletion',
```